### ISSUE TITLE
[TF2] Packed detail sprites blocked by sv_pure

### ISSUE THREAD

**Issue by aarmastah** [ValveSoftware/Source-1-Games#6721]
Detail sprites, notably defined in .vbsp files in the game's root directory, are incorrectly blocked by sv_pure when packed into custom maps. This is most noticeable in the set of community Jungle Inferno maps (cp_mossrock, plr_bananabay, pl_enclosure_final) and is easy to replicate.

![error](https://github.com/user-attachments/assets/d31eab34-3497-4d04-a748-953b4ed3029f)

1. Create a dedicated server, launched with the parameters "+map plr_bananabay +sv_pure 2"
2. Join the dedicated server 
3. Switch the map with "changelevel (any other map)"
4. Switch the map back to plr_bananabay with "changelevel plr_bananabay"

https://youtu.be/3rFUT20ouD8

Ideally, these vbsp files should be whitelisted from sv_pure when they are read from packed bsps. Alternatively, I found success placing the file in `vgui/replay/thumbnails` instead and compiling my own version of the map, as this directory is always whitelisted.

https://youtu.be/b87Hh-OXuJc

(This issue may be a duplicate of https://github.com/ValveSoftware/Source-1-Games/issues/2349)
